CHANGELOG.md

```diff
@@ -1,7 +1,34 @@
 # 0.6.0
 
+## Bug fixes
+
 * Fix panic when using incorrect globs f6or file and type selections.
 
+## New feautures
+
+* Also replace `Train-Case` (alo known as `Http-Case`) variants when using `--subvert`
+
+* The core functionnality has been re-written
+
+The output changes lightly:
+
+```
+# Using version < 0.6
+Patching foo/bar.js
+--- old is old
++++ new is new
+```
+
+
+```
+# Using version >= 0.6
+foo/bar.js 3: old is old
+foo/bar.js 3: new is new
```

Comment:
you're 100% right

Reply:
Oh also another thing I just noticed. In the pic above is says "Would performs 3 replacements", but I count 4 replacements of aaa -> bbb (in 3 lines, I think that's where the number comes from).

Reply:
Yeah. The message is not great.

I guess the Stats structs could hold 3 fields instead of two (`files_changed`, `lines_changed` and `total_replacements`) and then use total_replacements while displaying the stats at the end of main()